Source/Bifrost.Events.Azure.Tables/EventStore.cs

```diff
@@ -71,7 +71,7 @@ public void Commit(IEnumerable<EventAndEnvelope> eventsAndEnvelopes)
                 batch.Add(TableOperation.Insert(@event));
             });
 
-            _table.ExecuteBatchAsync(batch);
+            _table.ExecuteBatchAsync(batch).Wait();
```

Comment:
Do you see a reason why we would need to wait for this?

As I mentioned on the Issue #838